googlemock/include/gmock/gmock-spec-builders.h

```diff
@@ -1471,7 +1471,8 @@ class FunctionMockerBase : public UntypedFunctionMockerBase {
   virtual ~FunctionMockerBase()
         GTEST_LOCK_EXCLUDED_(g_gmock_mutex) {
     MutexLock l(&g_gmock_mutex);
-    VerifyAndClearExpectationsLocked();
+    if(!std::uncaught_exception())
```

Comment:
oh, yep. I missed it first time. Should I modify this fix using ```GTEST_GCC_VER_``` to support both versions of C++?